### ISSUE TITLE
FAILED 2025-04-27 snapshot stats and major changes

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-releases#128]
Stats are here: https://docs.google.com/spreadsheets/d/1tOQZmjzbUhqB9PUdeJ3gMsp-1-coKaQPJd_Dp763VXw/edit?gid=0#gid=0

**Comment by pgaudet** [comment]
Hi @kltm 
For mouse, we lost most EXP annotations: 

Staging: 

<img width="363" alt="Image" src="https://github.com/user-attachments/assets/0d7c6f7a-852f-4c95-83d7-df92e5fba403" />

AmiGO: 

<img width="392" alt="Image" src="https://github.com/user-attachments/assets/47413572-725f-4f4f-8863-ea6689a2c1a7" />

Should I investigate the upstream files? Or can you have a look ? 
Also this file https://mirror.geneontology.io/mgi-p2go-homology.gaf.gz is all scrambled for me: 

<img width="1442" alt="Image" src="https://github.com/user-attachments/assets/6b53760a-2cb6-4ab2-b265-ecc07ec91da4" />

----

Example of a mouse annotation by UniProt that we shouldn't have lost: PMID:33414192

https://amigo.geneontology.org/amigo/reference/PMID:33414192
https://amigo-staging.geneontology.io/amigo/reference/PMID:33414192

Thanks, Pascale

**Comment by pgaudet** [comment]
Other changes: 
- pombase: + 4203 >> 1 HDA paper
- rat + 191738 IEAs 


IEA Source | Diff
-- | --
Ensembl | 81800
UniProt | 75994
InterPro | 33156
GOC | -905
RGD | 0
RHEA | 1693

- ZFIN +26114 IEAs

IEA Source | Diff
-- | --
ZFIN | 1661
UniProt | 24050
RHEA | 403

**Comment by pgaudet** [comment]
Noting that FlyBase has removed their internal references, for example: 

AmiGO:
<img width="1271" alt="Image" src="https://github.com/user-attachments/assets/ee9a02ce-c14c-48f6-891a-ec798e262d40" />

Staging: 

<img width="1254" alt="Image" src="https://github.com/user-attachments/assets/4be41501-fd57-4d71-a198-4e94f3aae63c" />

**Comment by kltm** [comment]
Coming at it from species=mouse and experimental evidence, there is a large gap:

amigo-staging: 119842
current production: 179341

A ~60k difference loss on amigo-staging (snapshot).

Looking at the file differences:
```
sjcarbon@moiraine:/tmp$:) wc -l mgi-*
   340022 mgi-src-prod.gaf
   140669 mgi-src-snap.gaf
```
A loss of 200k likes right out the door sounds like a problem we need to deal with first.

Looking in the upstream source that we generate: last produced on April 24, 2025 and 1.3 MB compressed (would expect ~4.0 MB).

This file comes from the `p2go-homology-upstream-file-generator` pipeline. (Interestingly, it has a high failure rate.) On a successful rerun, the filesize is similarly small. This means that there is either an issue in the (unchanged and untouched) code, metadata, or the upstream files.

The download paths of the files used in this code are:

```
http://snapshot.geneontology.org/products/upstream_and_raw_data/mgi-src.gpi.gz
https://fms.alliancegenome.org/download/ORTHOLOGY-ALLIANCE-JSON_COMBINED.json.gz
http://skyhook.berkeleybop.org/p2go-homology-upstream-file-generator/products/upstream_and_raw_data/rgd-src.gaf.gz
http://skyhook.berkeleybop.org/p2go-homology-upstream-file-generator/products/upstream_and_raw_data/goa_human-src.gaf.gz
http://skyhook.berkeleybop.org/p2go-homology-upstream-file-generator/products/upstream_and_raw_data/goa_human_isoform-src.gaf.gz
https://www.informatics.jax.org/downloads/reports/HOM_MouseHumanSequence.rpt
http://ontology-build.geneontology.org/go.json
https://ftp.ebi.ac.uk/pub/databases/GO/goa/MOUSE/goa_mouse.gaf.gz
https://ftp.ebi.ac.uk/pub/databases/GO/goa/MOUSE/goa_mouse_isoform.gaf.gz
```

I cleaned out and re-ran (with some fixes https://github.com/geneontology/gopreprocess/issues/77) the generation pipeline, but the files are still coming up reduced.

We may have to dig in a bit more to try and figure out what exactly is wrong with the upstream processing. (I would also incidentally note I believe there has been an Alliance release.)

**Comment by pgaudet** [comment]
Hi @kltm 

> This file comes from the p2go-homology-upstream-file-generator pipeline. 

This would explain the issues with the ISO/IEA annotations, but we also had massive EXP changes, which should not be related to this, should it ?

**Comment by kltm** [comment]
@sierra-moxon Will help us dig into this a little.

**Comment by kltm** [comment]
@pgaudet There should now be a new `snapshot` to test on staging.

**Comment by pgaudet** [comment]
Mouse annotation decrease too large.
